src/config.h

```diff
@@ -0,0 +1,85 @@
+//
```

Comment:
Kind of a useless comment here, we don't really use this sort of comment

Reply:
a CMake automatic generated comment. will be removed.